AmplifyPlugins/Auth/Tests/AuthHostApp/AuthStressTests/AWSAuthBaseTest.swift

```diff
@@ -0,0 +1,132 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+@testable import Amplify
+import AWSCognitoAuthPlugin
+
+class AWSAuthBaseTest: XCTestCase {
```

Comment:
Can we reuse similar code from other scheme to avoid duplication?

Reply:
Fixed.